cache.rst

```diff
@@ -714,3 +714,119 @@ Clear all caches everywhere:
 .. code-block:: terminal
 
     $ php bin/console cache:pool:clear cache.global_clearer
+
+Encrypting the Cache
+--------------------
+
+To encrypt the cache using ``libsodium``, you can use the
```

Comment:
It think we should describe a little more exact what is encrypted, the cache value? the cache item? the complete storage?

Reply:
Good idea, I added a `note` about it.